### ISSUE TITLE
Sharing of my config file and lang file

### ISSUE THREAD

**Issue by AndiSlash0723** [25/EnchantmentsEnhance#120]
This is not a report.. only something that I didnt do in the past.
Which is config file and lang file sharing.
Recently came back to Minecraft and remember this.

Here is my config file and lang file (1.13 Minecraft, v7.2.0) :

[config and lang file.zip](https://github.com/25/EnchantmentsEnhance/files/3134908/config.and.lang.file.zip)

The lang file has translate to Traditional Chinese

In the config file, I did a little change to it :
1) I adjusted the enhance that an item can get when it reached specific level
For an example, Axe(tool) can only get lumberjack when it reached TET.

2) I adjusted the base chance, chance increase per failstack, maximum failstack
For example, I made +15's base chance and chance increase per failstack as same as BDO does to make getting high failstacks easier. Also, there is a higher base chance and chance increase per failstack than BDO for PEN attempt.

3) I changed the name from PRI to PEN base on TW server.
In TW Server : 
+16 = 長 = PRI
+17 = 廣 = DUO
+18 = 故 = TRI
+19 = 琉 = TET
+20 = 東 = PEN

4) I disabled force enhancement once the item reached +16
As long as I remember, the original config allows force enhancement.
But I disable this in my config just to make it feels more like BDO

5) More different item used as weapon.
In my config setting, I allowed enhancing Wait(Music disc), feather, stick etc.
But why? because I want to have more kind of weapons. For example, the Music Disc Wait is used as Dandelion Sah Chakram in my server, feather is used as Kutum Kunai and Nouver Kunai.

And... there are more changes in my config setting. Hope you guys will enjoy my config setting.

Also, I look forward to the 1.14 version of this plugin. Since 1.14 allows item to have multiple models, I can make it more close to the real BDO by giving them different models in the texture pack.
![image](https://user-images.githubusercontent.com/47967939/57025201-16b77b00-6c69-11e9-9517-b90b6ea75703.png)
Some of them are being rotated for better looking when players hold them.
For an example, the Dandelion Crescent Blade is actually a prismarine shard.

Thank you !

(((1.14版本的会出吗>< 我好期待呢!!

**Comment by lgou2w** [comment]
wow, amazing. What is the structure of the 1.14 custom model data, can you share it? The `LDK` used by `EE` is already compatible with the 1.14 server. As for when `EE` is compatible, it depends on the reply of @25 . XDDDDD

**Comment by AndiSlash0723** [comment]
Structure of the 1.14 custom model data?... I dont really get it but I followed this video to create them :
https://www.youtube.com/watch?v=tvL9Z0blPC8

This is for texture pack I think? What texture/model will an item get is kinda base on the setting in the texture pack.
For example, I made 8 selections for diamond swords.
If the diamond sword has a NBT tag "CustomModelData" and it's value is 1, base on my texture pack's setting, it will become Kzarka Shortsword (First slot in the chest). When the value is 2, it will become Kzarka Longsword (Second slot in the chest) and so on.

Btw I putted the plugin into the 1.14 server, it gave me some 2 errors when I click on empty slots (no matter my inventory or the menu). Here is the link :
https://imgur.com/a/32maZtC
Not sure what happened, I had Vault in my server... Is it because the latest version of Vault doesn't support 1.14? @@
I will open a new issue to report this if required.

Thank you
